From 4def0c681a537feee886b93ab954057c06ffffb9 Mon Sep 17 00:00:00 2001
From: Yukang Zhong <yukang.zhong@amlogic.com>
Date: Thu, 25 Jun 2015 10:40:07 +0800
Subject: [PATCH 5666/5965] PD#108115: dtd: n301: fix fe tuner pinmux conflict

This commit will fix i2c r/w error. But even can't auto search
for dtv. the problem is tracked by another bugid.

Change-Id: Iaf53084cb950895b8f96ee4aabb8830805097fc8
---
 arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
index 7080eece1174..a87261af236d 100644
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n301.dtd
@@ -1722,7 +1722,7 @@
 		atv_demod0 = "aml_atv_demod";
 		fe0_atv_demod = <0>;
 		pinctrl-names = "default";
-		pinctrl-0 = <&dvb_fe_pins>;
+		//pinctrl-0 = <&dvb_fe_pins>;
 	};
 
 	/*smartcard {
-- 
2.19.0

